docs/docker_container.md

```diff
@@ -323,6 +323,10 @@ OpenVINO Model Server, starting from release 2021.1, monitors the changes in its
 
 - In case the new config.json is invalid (not compliant with json schema), no changes will be applied to the served models.
 
+- Config reload may be also triggered using [Config Reload API](./model_server_rest_api.md#config-reload).
```

Comment:
I think this chapter requires some refactoring:
After the intro in this chapter add info that config changes can be detected regularly with a frequency defined in --file_system_poll_wait_seconds parm or can be triggered via `config' REST API calls.
Summarize the behaviour in points:
- remove models
- adding models
- updating models configs
next explain error handling in points:
- invalid config format that fail json schema - no changes applied
- incorrect parameters for individual models and pipelines - applied changes to correctly configured models and pipelines